plugins/aea-cli-ipfs/aea_cli_ipfs/core.py

```diff
@@ -39,10 +39,9 @@
 
 @click.group()
 @click.pass_context
-@click.option("--offline", is_flag=True, default=False)
```

Comment:
Why do we no longer provide this option?

Reply:
I thought we won't be using local IPFS daemons anymore, But that's not the case anymore I'll revert the changes